### DISCUSSION TITLE
Where to find `gfm` option in remark 12.x? (for MDX 1.x usage)

### DISCUSSION THREAD

**Discussion by lunelson** [remarkjs/remark#657] [Q&A] [answered]
At the repository for [remark-gfm](https://github.com/remarkjs/remark-gfm) it says "while you're still on remark 12, use the `gfm` option for remark". I need to use remark 12 because I am using MDX 1.x; but I can't find documentation of this option anywhere. Can someone explain how it is used?

**Comment by wooorm** [comment, marked as answer]
As we’re using Git, you can browse past documentation: https://github.com/remarkjs/remark/tree/12.0.1

**Reply by lunelson** [reply]
Yes, I tried that first, but couldn't find any mention of it. On the main README, "Documentation" is referred to the unified.js site. However [I see now it's an option in `remark-parse`.](https://github.com/remarkjs/remark/blob/12.0.1/packages/remark-parse/readme.md#api)
